src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java

```diff
@@ -43,7 +43,7 @@
       effectTags = {OptionEffectTag.UNKNOWN},
       help =
           "Specify how spawn actions are executed by default."
-              + "'standalone' means run all of them locally."
+              + "'standalone' means run all of them locally and it disables sandboxing for tests."
               + "'sandboxed' means run them in namespaces based sandbox (available only on Linux)"
```

Comment:
This is also wrong by now, it should read:

'sandboxed' means to run them in a sandboxed environment with limited privileges (details depend on platform support)